client/state/jetpack-connection-health/selectors/get-jetpack-connection-health.js

```diff
@@ -0,0 +1,16 @@
+import 'calypso/state/jetpack-connection-health/init';
+
+/**
+ * Returns true if the current site Jetpack connection was checked and is unhealthy
+ * @param  {Object}  state         Global state tree
+ * @param  {number}  siteId        Site ID
```

Comment:
If the site ID is always a number, why do we need the `siteId` check below? Is there a chance we'll have `siteId` of `0`?